<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseLogin</class>
 <widget class="QWidget" name="DatabaseLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ico_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="minButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Warnning">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="adm">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>adm</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="stu">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>stu</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="tea">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>tea</string>
   </property>
  </widget>
  <zorder>labelTitle</zorder>
  <zorder>closeButton</zorder>
  <zorder>minButton</zorder>
  <zorder>labelLogin</zorder>
  <zorder>usernameEdit</zorder>
  <zorder>loginBtn</zorder>
  <zorder>passwordEdit</zorder>
  <zorder>ico_label</zorder>
  <zorder>Warnning</zorder>
  <zorder>adm</zorder>
  <zorder>stu</zorder>
  <zorder>tea</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
